CMake/AbseilHelpers.cmake

```diff
@@ -62,7 +62,107 @@ function(absl_library)
   endif()
 endfunction()
 
+#
+# CMake function to imitate Bazel's cc_library rule.
```

Comment:
you should add a link to https://docs.bazel.build/versions/master/be/c-cpp.html#cc_library
Usually CMake user don't use bazel

Reply:
Technically this `absl_cc_library` is not for use of normal CMake users. The hope is to create CMake functions that close match the Bazel rules so that we can generate CMakeFiles.txt from script without human intervention (so that Abseil team don't need to go and learn CMake). It is implementation detail that CMake user as a consumer of Abseil library does not need to know.